packages/modelviewer.dev/src/docs-and-examples/sidebar.ts

```diff
@@ -0,0 +1,225 @@
+/* @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let globalCurrentView: string[] = [];
+let previouslyActive: string = '';
+let toRemove = '';
+let order: any[] = [];
+
+let isFirstOpen = true;      // is true on the first observation of all entries
+let everyEntry: any[] = [];  // a list of all attributes/properties etc.
+
+interface SidebarIds {
+  name: string, subcategory: string, category: string
+}
+
+function activateSidebar(
+    sidebarName: string, sidebarSubcategory: string, sidebarCategory: string) {
```

Comment:
You should just take a single `SidebarIds` argument here and below.

Reply:
Yeah, that was silly of me not to do